components/gems/Gemfile.lock

```diff
@@ -321,7 +325,7 @@ GEM
       concurrent-ruby (~> 1.0)
       ffi-yajl (~> 2.2)
       http (~> 4)
-    chef-utils (15.10.12)
+    chef-utils (16.0.287)
     chef-vault (4.0.1)
```

Comment:
There's a chef-vault dep in the gemfile that can go away now that it's a dep of chef itself